cfg/rh-0.7/config.yaml

```diff
@@ -10,12 +10,14 @@ master:
   scheduler:
     bins:
       - "openshift start master controllers"
+      - "hyperkube kube-scheduler"
```

Comment:
@robertojrojas this should probably be hybershift instead of hyperkube, shouldn't it?